### PR TITLE
release(v1.9.0-alpha.1): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v1.9.0-alpha.1 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,557 @@
...
+
+
+### udevd
+
+Talos previously used `udevd` to provide `udevd`, now it uses `systemd-udevd` instead.
+
+
+### Component Updates
+
+Linux: 6.6.59
```
Comment (frezbo) [inline CHANGELOG.md:46]: this is now rendered as a single line, should have been a list

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,557 @@
...
+### `talosctl cgroups`
+
+The `talosctl cgroups` command has been added to the `talosctl` tool.
+This command allows you to view the cgroup resource consumption and limits for a machine, e.g.
+`talosctl cgroups --preset memory`.
+
+
+### udevd
+
+Talos previously used `udevd` to provide `udevd`, now it uses `systemd-udevd` instead.
```
Comment (frezbo) [inline CHANGELOG.md:41]:
```suggestion
Talos previously used `eudevd` to provide `udevd`, now it uses `systemd-udevd` instead.
```

**Review by frezbo — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (smira) [reply CHANGELOG.md:46]: Yep, let's fix it in main for the next bump

File: CHANGELOG.md (hunk 2 above)
Comment (smira) [reply CHANGELOG.md:41]: Let's do all fixes in `main`, not on the release time ;) I agree with all of that, but it'd be easier to do that before we get to the release.

Comment (smira) [reply CHANGELOG.md:41]: I pushed fixes to my next PR

**Pull request merged**